packages/stockflux-watchlist/src/components/watchlist/Watchlist.js

```diff
@@ -59,6 +65,33 @@ const Watchlist = () => {
     'stockflux-watchlist'
   );
 
+  useEffect(() => {
+    if (notification.state === 'LAUNCHED') {
+      const alreadyInWatchlist = watchlist.includes(newSymbol);
+      notification.populate(
+        <>
+          <div className="notification-icon">
+            <img
+              id="icon"
+              className={alreadyInWatchlist ? 'arrow-up' : 'card-icon'}
+              src={alreadyInWatchlist ? 'ArrowUp.png' : 'CardIcon.png'}
+              alt="Notification Icon"
+            />
+          </div>
+          <div id="notification-content">
+            <p id="title">WATCHLIST UPDATED</p>
+            <hr />
+            <p id="info">
+              <span id="symbol">{newSymbol}</span>
+              <span id="message">{`${
+                alreadyInWatchlist ? ' moved' : ' added'
+              } to the top`}</span>
```

Comment:
If I add something new to the watchlist, I get the message 'moved to the top', even if the list is empty. Should it be 'added to the top'? Also when I add the same item twice the notification fires twice. Is this the correct behaviour?

Reply:
Yeah, that is expected behavior (firing twice). Messages are going to change either way in the Notification UX refactor PR.